0004-siren-prototype-accept-decline.adoc

```diff
@@ -0,0 +1,90 @@
+= Siren Prototype for comit-rs HTTP API - Accept/Decline actions
+Franck Royer <franck.royer@coblox.tech>
+:toc:
+:revdate: 2019-05-02
+
+* Author(s): {authors}
+* Date: {revdate}
+
+Tracking issue: https://github.com/comit-network/comit-rs/issues/917[comit-rs#917]
+
+== Context
+
+In order to move forward easily with Bobtimus prime we should define the siren json document of how a IN_PROGRESS response look like in each state of accept and decline, i.e. when you call GET /swaps/ for Bob.
+
+Siren repository: https://github.com/kevinswiber/siren/
+
+The prototype must conform to https://github.com/kevinswiber/siren/blob/master/siren.schema.json[Siren JSON schema]
+
+== Research
+
+.The prototype
+[source,json]
+----
+{
+  "class": ["swaps"],
+  "entities": [
+    {
+      "class": ["swap"],
+      "rel": ["item"],
```

Comment:
Where does this point to? What is "item" ?

Reply:
Please have a look at the previous discussion in this PR:
https://github.com/comit-network/spikes/pull/7#discussion_r280271091